src/server.cpp

```diff
@@ -684,12 +684,6 @@ void CServer::OnAboutToQuit()
 
     Stop();
 
```

Comment:
Was it deliberate to remove the `if ( GetServerRegistered() ) { Unregister(); }` here? I can't see it has moved to anywhere else.

Reply:
9222c202158deff4b18f75975355f931a9dbaa18 was the original change doing this and `ServerListManager.SetDirectoryType()` seems to handle `Unregister()` if necessary: https://github.com/jamulussoftware/jamulus/blob/master/src/serverlist.cpp#L269-L275

It's called in https://github.com/jamulussoftware/jamulus/pull/2199/files#diff-c044ba02ccf7862f1e2f56436d5b92baa7bb35601791846655fb07ac253a8697R951

So this looks fine to me.